### PR TITLE
obsoleting 7500033 and 7500034

### PR DESCRIPTION
**Pull request by megbalk**
issue #2646

### CODE REVIEW

**Comment by megbalk** [comment]
@shawntanzk should I assign a different reviewer?

**Comment by shawntanzk** [comment]
oh im sorry - totally slipped my radar, thanks for pinging
```
 :: ERROR: logic-axiom-references-obsolete: UBERON:7500033 has relationship to https://orcid.org/0000-0003-2699-3066
  :: ERROR: logic-axiom-references-obsolete: UBERON:7500034 has relationship to https://orcid.org/0000-0003-2699-3066
```

**Comment by shawntanzk** [comment]
argh it should be an annotation not a logical axiom - not sure why this is happening, might be obo format stuff? @matentzn is that the case?

**Comment by matentzn** [comment]
Is it a logical axiom? This could simply be a a problem with the QC check.

- Quick fix: remove the dc contributor annotations from the obsolete classes
- Correct fix: Fix the failing check in the makefile to allow contributor relationships (I am assuming that `contributor` has `metadata: True` in the OBO file.

**Comment by megbalk** [comment]
I can remove contributor if that helps? Luckily because I have an id range, one could figure out who made the terms in the first place?

**Comment by shawntanzk** [comment]
Yeah for now I would remove the contributor - ill write a ticket to fix the QC check

**Comment by megbalk** [comment]
@shawntanzk fixed, let me know if it looks ok.

**Comment by shawntanzk** [comment]
```
 :: ERROR: logic-axiom-references-obsolete: UBERON:7500034 has relationship to https://orcid.org/0000-0003-2699-3066
```

**Comment by megbalk** [comment]
I wasn't working on 7500034...why is that popping up as an error now?

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -221904,19 +221904,20 @@ relationship: part_of UBERON:6001767 ! insect antennal disc
...
+is_obsolete: true
+replaced_by: UBERON:0009984
 
 [Term]
 id: UBERON:7500034
-name: distal-most point of trochlea of humerus
-def: "The distal-most point of the trochlea of a humerus." [https://orcid.org/0000-0003-2699-3066]
-intersection_of: UBERON:0006983 ! anatomical point
-intersection_of: distalmost_part_of UBERON:0000144 ! trochlea of humerus
+name: obsolete distal-most point of trochlea of humerus
```
Comment (shawntanzk) [inline src/ontology/uberon-edit.obo:221915]: looks like you did obsolete UBERON:7500034

Comment (megbalk) [reply src/ontology/uberon-edit.obo:221915]: oops! ok I'll fix it.

**Review by shawntanzk — APPROVED** [review]

**Comment by cmungall** [comment]
checks pass, 1 approval, go ahead and merge!

I think in general we should be more precise about reasons - when is too specific too specific? But as the obsoleter is the original requester let's just merge!

**Pull request merged**